core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -265,10 +265,10 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     // get chosen before the noPref task
     assert(manager.resourceOffer("exec1", "host1", NODE_LOCAL).get.index == 2)
 
-    // Offer host2, exec3 again, at NODE_LOCAL level: we should choose task 2
+    // Offer host2, exec2, at NODE_LOCAL level: we should choose task 3
```

Comment:
The resource offering is host = `host2` and executor = `exec2`. So the comment should reflect this offering. 

The `index` is actually the index of tasks in the FakeTaskSet. Let me double check whether I should change this line or the previous example.